content/applications/finance/accounting/customer_invoices/terms_conditions.rst

```diff
@@ -31,3 +37,9 @@ content, and click :guilabel:`Save`. The link to that page is then added as a no
 
 .. image:: terms_conditions/terms-webpage.png
    :alt: Example of terms and conditions as a web page
+
+.. tip::
+   This terms and conditions page can be displayed in the footer of all documents. To do so, open
+   the database :guilabel:`Settings` and click :guilabel:`Configure Document Layout` under the
+   :guilabel:`Companies` section. Add the link to the web page under the :guilabel:`Footer` field
+   and :guilabel:`Save`.
```

Comment:
I think that's the old workaround before there was a default page directly within this feature.
So that tip is only valid up to 14.0.